src/main/java/com/mailjet/client/transactional/response/MessageResult.java

```diff
@@ -0,0 +1,44 @@
+package com.mailjet.client.transactional.response;
+
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class MessageResult {
+
+    /**
+     * Indicates the sending status of the message.
+     * Possible values:
+     * success
+     * error
+     */
+    private String status;
```

Comment:
would it be possible to use an enum for this?

Reply:
nice catch!

Reply:
Added enum